northstar/src/runtime/state.rs

```diff
@@ -36,61 +30,56 @@ use npk::{
 };
 use std::{
     collections::{HashMap, HashSet},
-    fmt,
     path::{Path, PathBuf},
     result,
     sync::Arc,
 };
 use tokio::{sync::oneshot, task, time};
 
 #[derive(Debug)]
-pub(super) struct State<'a> {
+pub(super) struct State<'a, L>
```

Comment:
would be nice to get rid of the lifetime here as well

Reply:
Why do you want to remove the lifetime? It's just used for the ref of the config to avoid a copy.